aws/export_test.go

```diff
@@ -5,6 +5,25 @@ import (
 	"time"
 )
 
+// Auth:
+// Exporting methods for testing
+
+func (a *Auth) AccessKey() string {
```

Comment:
Oh I think govet was unhappy with the implicit copy given the object has a mutex member.